frontend/src/pages/NewFile/components/ModuleCard/ModuleCard.tsx

```diff
@@ -0,0 +1,55 @@
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+
+import { Logo } from '/src/components'
+
+import { MoreVertical, Trash } from 'lucide-react'
+import { ButtonHTMLAttributes, Ref } from 'react'
+import { CardContainer, CardDetail, CardImage, SelectedTemplateOverlay, TitleWithAction } from './moduleCardStyle'
+dayjs.extend(relativeTime)
+
+type moduleCardProps = {
+    name: string
+    // type?: ProjectType | '???' // '???' is used has a default type
```

Comment:
```suggestion
```
Though maybe a basic card component would be in order so the design with project card doesn't go out of sync